indico/modules/events/registration/templates/management/regform_list.html

```diff
@@ -81,7 +81,11 @@
                         <div class="group">
                             <a href="{{ url_for('.manage_reglist', regform) }}" class="i-button icon-users">
                                 {% trans %}Registrations{% endtrans %}
-                                <span class="badge">{{ registration_counts.get(regform.id, 0) }}</span>
+                                <span class="badge">{{ registration_counts.get(regform.id, 0) }}
```

Comment:
I would move the Jinja tag down, in order to keep indentation consistent. if you're just trying to avoid a whitespace in the HTML, use `{{-` or `{#- -#}`.